src/pages/HomeAdministrator.vue

```diff
@@ -12,7 +12,16 @@
               <div class="text-md text-gray-500 ml-6">Lists administrations assigned to your account</div>
             </div>
             <div class="flex align-items-center gap-2">
-              <div class="flex gap-3 align-items-center justify-content-start">
+              <div class="flex gap-3 align-items-stretch justify-content-start">
+                <div v-if="isSuperAdmin" class="flex flex-column gap-1">
+                  <small class="text-gray-400">Show test administrations</small>
+                  <PvToggleButton
+                    v-model="fetchTestAdministrations"
+                    :on-label="showTestAdminsOnLabel"
+                    off-label="Hiding"
+                    class="p-2 rounded align-self-center my-auto"
+                  />
+                </div>
```

Comment:
Unfortunately there is no toggle switch in PrimeVue 3, so I went with the toggle button for now until we are ready to upgrade.

Reply:
Are any other LEVANTE checks necessary? Otherwise I will merge.

Reply:
Nevermind, I was looking at the wrong docs. They changed it from Input Switch to Toggle Switch in V4 of Primevue.

Are there any othe LEVANTE checks that are needed? Otherwise I will merge.